patricktunez123/Andela_Teamwork
# #168760540 Add CSS to fix UI colors and dead links issues

#### What does this PR do?
Remove mixture of colors and use one color
#### Description of Task to be completed?
Have all UI pages with the same color and font
#### How should this be manually tested?
In URL type https://patricktunez123.github.io/Andela_Teamwork/UI/ 
click login button you should see different boring colors
#### What are the relevant pivotal tracker stories?
#168760540
